plugins/modules/ec2_vpc_egress_igw_info.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+DOCUMENTATION = r"""
+---
+module: ec2_vpc_egress_igw_info
```

Comment:
I'm not sure if this module should be in the same PR. I guess we should only include the modules we'd want to migrate from c.aws to a.aws.

Reply:
I'd suggest moving this module into a separate PR. We'll merge it first so there would not be any issue with the integration tests.